ci-operator/config/openstack-k8s-operators/nova-operator/openstack-k8s-operators-nova-operator-master.yaml

```diff
@@ -1,14 +1,18 @@
 base_images:
+  cli:
+    name: "4.10"
+    namespace: ocp
+    tag: cli
   golangci-lint:
     name: golangci-lint
     namespace: ci
     tag: v1.46.0
-binary_build_commands: make build
+  opm-builder:
+    name: opm-builder
+    namespace: ci
+    tag: latest
 build_root:
-  image_stream_tag:
-    name: release
-    namespace: openshift
-    tag: rhel-8-release-golang-1.18-openshift-4.12
+  from_repository: true
 images:
```

Comment:
Add below under images to do the image build:

~~~
- dockerfile_literal: |
    FROM src
    RUN pip3 install python-openstackclient
    COPY oc /usr/bin/oc
    COPY opm /usr/bin/opm
  from: src
  inputs:
    cli:
      paths:
      - destination_dir: .
        source_path: /usr/bin/oc
    opm-builder:
      paths:
      - destination_dir: .
        source_path: /bin/opm
  to: openstack-operator-ci-build-deploy
~~~

Reply:
Done